plugins/modules/btrfs_subvolume.py

```diff
@@ -0,0 +1,672 @@
+#!/usr/bin/python
+
+# Copyright (c) 2022, Gregory Furlong <gnfzdz@fzdz.io>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: btrfs_subvolume
+short_description: Creates/update/delete btrfs subvolumes
+version_added: "1.0.0"
+
+description: Creates, updates and deletes btrfs subvolumes based on the provided parameters.
+
+options:
+    automount:
+        description:
+        - Allow mounting the targeted btrfs filesystem as necessary to validate the current state and make required changes.
+        required: false
+        type: bool
+        default: false
+    default:
+        description:
+        - Make the targetted subvolume the default subvolume
```

Comment:
```suggestion
        - Make the targetted subvolume the default subvolume.
```
All descriptions must contain complete sentences.